lib/inspec/resources/users.rb

```diff
@@ -715,6 +720,7 @@ def collect_user_details # rubocop:disable Metrics/MethodLength
             home = $_.HomeDirectory[0]
             shell = $null
             domain = $Computername
+            lastlogin = if($_.lastlogin.getType().Tostring() -eq "System.Management.Automation.PSMethod" ){ $null }else{[String]$_.lastlogin}
```

Comment:
There has to be a better way to return only a date or $null.

Reply:
maybe! I'm going to have to dig into this one =)